adb-server/adb-server-command-types/src/main/java/com/kaspersky/adbserver/commandtypes/AdbCommand.kt

```diff
@@ -2,4 +2,4 @@ package com.kaspersky.adbserver.commandtypes
 
```

Comment:
General feedback.

I guess we can use `Runtime::exec(String[] cmdarray)` instead of `Runtime::exec(String command)` in `CmdCommandPerformer.perform`. Have a look at `java.lang.Runtime`, where both methods call the same method and `command` param in `Runtime::exec(String command)` is parsing into the array of strings.

I don't support introducing separate classes like `ComplexAdbCommand`. I think we need to add the new constructors like `Command(command: String, arguments: List<String>)` and the correspondent methods in `AdbTerminal` and etc.

I don't support syntaxis like `(command: List<String>)` and `(command): String`. It's confusing. Also, the most popular syntaxis in these cases is `(command: String, arguments: List<String>)`. Yes, another command is an argument of the main command. Cases like `sh -c "adb shell dumpsys deviceidle | grep mForceIdle"` should be described in the documentation and examples.